packages/go_router/lib/src/delegate.dart

```diff
@@ -233,7 +233,7 @@ class GoRouterDelegate extends RouterDelegate<RouteMatchList>
           exitingMatches.length - 1,
           context: navigatorContext,
           matches: exitingMatches,
-          configuration: configuration,
+          // configuration: configuration,
```

Comment:
?!

Reply:
Oh my ! Sorry about that. I removed it in [refactor: Remove commented code](https://github.com/flutter/packages/pull/6623/commits/560d3f44054795cc828101f3ad889830979d9853)